specification/api-tracing.md

```diff
@@ -397,7 +410,8 @@ with the moment when they are added to the `Span`.
 An `Event` is defined by the following properties:
 
 - (Required) Name of the event.
-- (Optional) One or more `Attribute`.
+- (Optional) One or more `Attribute`s with the same semantics as defined for
+  [Span Attributes](#set-attributes).
```

Comment:
What is the intended behavior if we have something as follow?
```java
span.AddEvent("Event Name", Attribute{key: "Key", value: null})
```

Reply:
Will be the event dropped or will it be register without any attribute?

Reply:
same as `span.AddEvent("Event Name", null)` == `span.AddEvent("Event Name")`